client/state/selectors/test/get-application-passwords.js

```diff
@@ -0,0 +1,45 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { getApplicationPasswords } from 'state/selectors';
+
+describe( 'getApplicationPasswords()', () => {
+	test( 'should return application passwords of the current user', () => {
+		const appPasswords = [
+			{
+				ID: 12345,
+				name: 'Example',
+				value: '2018-01-01T00:00:00+00:00',
+			},
+			{
+				ID: 23456,
+				name: 'Test',
+				value: '2018-02-01T08:10:00+00:00',
+			},
+		];
+		const state = {
+			applicationPasswords: {
+				items: appPasswords,
+			},
+		};
+		const result = getApplicationPasswords( state );
+		expect( result ).toEqual( appPasswords );
```

Comment:
Could be `toBe`.

Reply:
https://github.com/Automattic/wp-calypso/pull/23906/commits/e41f8c698b858a3379b035541852db9db4f950a7